### PR TITLE
Deprecate editor support (for srclib.org static site / documentation)

### PR DESCRIPTION
**Pull request by rothfels**
Precursor to `nodb` -> `master` merge

@beyang

### CODE REVIEW

File: docs/sources/plugins/compatiblebinaries.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Editor plugins are temporarily incompatible with srclib `master`.
+
+To use editor plugins, please download one of the following `srclib` binaries:
+
+* [Linux amd64](https://api.equinox.io/1/Applications/ap_BQxVz1iWMxmjQnbVGd85V58qz6/Updates/Asset/srclib.zip?os=linux&arch=amd64&channel=stable)
+* [Linux i386](https://api.equinox.io/1/Applications/ap_BQxVz1iWMxmjQnbVGd85V58qz6/Updates/Asset/srclib.zip?os=linux&arch=386&channel=stable)
+* [Mac OS X](https://api.equinox.io/1/Applications/ap_BQxVz1iWMxmjQnbVGd85V58qz6/Updates/Asset/srclib.zip?os=darwin&arch=amd64&channel=stable)
+
+Once downloaded, unzip and place the `src` program in your `$PATH`.
```
Comment (beyang) [inline docs/sources/plugins/compatiblebinaries.md]: s/src/srclib/

File: docs/theme/home.html [hunk 2]
```diff
@@ -70,21 +70,17 @@
...
-        </button>
+<!--    <div class="row"> -->
+<!--      <div class="col-md-12"> -->
+<!--        <p> -->
+<!--        <a href="/plugins/emacs"><button class="btn btn-primary"><img style="height: 1em;" src="/images/editors/emacs.svg"> Emacs</button></a> -->
+<!--        <a href="/plugins/sublimetext"><button class="btn btn-primary"><img style="height: 1em;" src="/images/editors/sublime.png"> Sublime</button></a> -->
+<!--        <a href="/plugins/atom"><button class="btn btn-primary"><img style="height: 1em;" style="height: 1em;" src="/images/editors/atom.png"> Atom</button></a> -->
+<!--        <button data-toggle="popover" data-placement="top" data-content="Vim support is not yet implemented" type="button" class="btn btn-default btn-disabled"> -->
+<!--          <img class="desaturate" style="height: 1em;" src="/images/editors/vim.svg"> Vim -->
+<!--        </button> -->
```
Comment (beyang) [inline docs/theme/home.html]: Delete instead of commenting out.

File: docs/theme/home.html [hunk 3]
```diff
@@ -96,14 +92,14 @@
...
+<!--        <p> -->
+<!-- -->
+<!--        <p>Interested in building a plugin for an editor srclib doesn't yet -->
+<!--        support? <a target="_blank" href="https://twitter.com/srclib">Let us -->
+<!--        know</a>&mdash;we'd love to help! -->
+<!-- -->
+<!--      </div> -->
+<!--    </div> -->
 
     <!-- <div class="row"> --
```
Comment (beyang) [inline docs/theme/home.html]: you can also delete this.

**Comment by beyang** [comment]
lgtm after comments

File: docs/theme/home.html (hunk 2 above)
Comment (rothfels) [inline docs/theme/home.html]: Other commented out code in this file...shall I remove that as well?

File: docs/theme/home.html (hunk 3 above)
Comment (rothfels) [inline docs/theme/home.html]: :+1:

there's quite a bit more, I'll get rid of all of it

**Pull request merged**